src/en/controllers-creating.md

```diff
@@ -7,20 +7,34 @@ TODO: Review again soon (created: March 2016)
 Use the `juju bootstrap` command to create a controller (and model) for a given
 cloud provider:
 
-`juju bootstrap <controller name> <provider name> [--options]`
+`juju bootstrap [options] <controller name> <provider name>`
 
 See `juju help bootstrap` for details on this command or see the
 [command reference page](./commands.html#juju-bootstrap).
 
 
+## Notes
+
+The '--upload-tools' option is used to make agent software available that is
+more recent than the default binary. This is done when some features may not
+yet be compiled in to the agent for the Ubuntu release being installed. Note
+that Juju will default to the latest LTS (see `distro-info --lts` command).
+
+
```

Comment:
I'll reword the Note so that we won't have to remove anything. Surely this will always be an issue. I'll remove the option from the examples.